lib/turn.rb

```diff
@@ -0,0 +1,26 @@
+
+
+class Turn
+  attr_reader :guess,:card
+
+  def initialize(guess,card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if @guess == card.answer
+    true
+      else
+        false
+      end
+  end
+
+  def feedback
+    if correct? == true
```

Comment:
This line could say `if correct?` since `correct?` will either be `true` or `false`.